src/index.ts

```diff
@@ -96,25 +97,65 @@ class DataMaker {
       Authorization: `${this.apiKey}`,
       ...this.options.defaultHeaders,
     };
-  }
-
+  };
+  /**
+   * Generate data from custom template.
+   * @param template 
+   * @returns 
+   */
   async generate(template: Template) {
     if (!template) {
       throw new Errors.DataMakerError(
-        "You must provide a template to generate data"
+        "You must provide a template to generate data."
       );
-    }
+    };
 
     if (!template.quantity) {
       template.quantity = 1;
-    }
+    };
+
+    return (await fetchDatamaker(this.options.baseURL, this.headers, template)).json(); 
+  };
+  /**
+   * Generate data using template from you Datamaker account. As arguments provide ID of a template from your account and a number of entries to be generated.
+   * Requires Datamaker api key to be defined in your project.
+   * @param templateId 
+   * @param quantity 
+   * @returns 
+   */
+  async generateFromTemplate(templateId: string, quantity: number = 1) {        
```

Comment:
Can we rename this to `generateFromTemplateId` - From template is a bit confusing. Given that the `generate` function generates from a template.

Reply:
Other than the above, this PR looks good. Thanks @lhotovy

Reply:
@RatulMaharaj sure, will do that